>From b2ca810a5186063cdb749cdb78826246fa2a8eb5 Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Thu, 18 Jun 2015 13:02:58 +0200
Subject: [PATCH] Exclude .devhelp2 files from being compressed

gtkdoc reads the .devhelp2 files to create proper cross-references in
the HTML documentation. Compressing those files breaks this since gtkdoc
does not support reading compressed files.

Since those .devhelp2 files are usually rather small and only installed
on developer machines, the space saveings are not worth it.

Thus exclude the files from being compressed by dh_compress.

https://bugs.launchpad.net/ubuntu/+source/gtk-doc/+bug/1466210
---
 dh_compress | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dh_compress b/dh_compress
index eb094a4..f33748c 100755
--- a/dh_compress
+++ b/dh_compress
@@ -124,6 +124,7 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 				! -iname "*.xz" ! -iname "*.lz" ! -iname "*.lzma" \\
 				! -iname "*.svg" ! -iname "*.svgz" ! -iname "*.js" \\
 				! -name "index.sgml" ! -name "objects.inv" ! -name "*.map" \\
+				! -name "*.devhelp2" \\
 				! -name "copyright" 2>/dev/null || true;
 			find usr/share/fonts/X11 -type f -name "*.pcf" 2>/dev/null || true;
 		`);
-- 
2.1.4

